src/metabase/models/pulse.clj

```diff
@@ -240,20 +240,20 @@
   ([{:keys [archived? dashboard_id]
      :or   {archived?    false
             dashboard_id nil}}]
-   (let [base-query {:select    [:p.*]
+   (let [base-query {:select    [:p.* [:%lower.p.name :lower-name]]
                      :modifiers [:distinct]
                      :from      [[Pulse :p]]
                      :where     [:and
                                  [:= :p.alert_condition nil]
                                  [:= :p.archived archived?]]
-                     ;; :order-by  [[:%lower.name :asc]]
-                     }
+                     :order-by  [[:lower-name :asc]]}
```

Comment:
@robdaemon I assume this was commented out since it didn't work without the `select` hack. If there was a real reason for commenting out (or if you have a better hack suggestion—I'm not thrilled with the need for `dissoc`) let me know